raiden/network/rpc/client.py

```diff
@@ -938,7 +938,10 @@ def settled(self):
     def close(self, our_address, their_transfer):
         """`our_address` is an argument used only in mock_client.py but is also
         kept here to maintain a consistent interface"""
-        their_encoded = their_transfer.encode()
+        their_encoded = ''
```

Comment:
I am a fan of the ternary operator and I would use it here. Not a requirement, just a suggestion.